Jenkinsfile_CNP

```diff
@@ -23,6 +23,7 @@ def secrets = [
         secret('ccd-api-gateway-oauth2-client-secret', 'OAUTH2_CLIENT_SECRET'),
         secret('ccd-befta-caseworker-3-pwd', 'CCD_BEFTA_CASEWORKER_3_PWD'),
         secret('ccd-befta-master-caseworker-pwd', 'CCD_BEFTA_MASTER_CASEWORKER_PWD'),
+        secret('TEST-DATA-RELOAD-FREQUENCY', 'TEST_DATA_RELOAD_FREQUENCY'),
```

Comment:
This would be just a simple environment variable with an open value.
Moreover, we even wouldn't need to have one at all, as the default value is 10 minutes, which will make it for our purpose. Maybe you can consider setting it to a high value like 100 minutes anyway, which will mean effectively, always reuse data from smoke tests.